snowplow_tracker/tracker.py

```diff
@@ -122,25 +122,27 @@ def get_timestamp(tstamp: Optional[float] = None) -> int:
     Tracking methods
     """
 
-    def track(self, pb: payload.Payload) -> "Tracker":
+    def track(self, pb: payload.Payload) -> str:
```

Comment:
There is an if condition before returning the ID, so I think this should be optional:

```suggestion
    def track(self, pb: payload.Payload) -> Optional[str]:
```